FluentDarkModeKit.xcodeproj/project.pbxproj

```diff
@@ -753,6 +753,7 @@
 				DYLIB_COMPATIBILITY_VERSION = 1;
 				DYLIB_CURRENT_VERSION = 1;
 				DYLIB_INSTALL_NAME_BASE = "@rpath";
+				GCC_TREAT_WARNINGS_AS_ERRORS = YES;
```

Comment:
We don't need this for Debug as this may be annoying when developing.

Reply:
makes sense, will limit to release only